app/models/project.rb

```diff
@@ -8,15 +8,15 @@ class Project < ApplicationRecord
 
   has_many :collaborations, dependent: :destroy
   has_many :collaborators, source: 'user' , through: :collaborations
-  has_many :taggings
+  has_many :taggings, dependent: :destroy
   has_many :tags, through: :taggings
   mount_uploader :image_preview, ImagePreviewUploader
 
   self.per_page = 8
 
   acts_as_commontable
   # after_commit :send_mail, on: :create
-
+  scope :public_view, -> { where(project_access_type: "Public") }
```

Comment:
just `public` is enough

Reply:
rails 4.1 doesn't allow the use of reserved words like public, private anymore.

Reply:
My bad!. Can we have better name then? What about `open` ?

Reply:
``` open ``` seems nice :wink: